site/docs/components/stepped-tracker/accessibility.mdx

```diff
@@ -8,4 +8,6 @@ data:
   $ref: ./#/data
 ---
 
-Stepped Tracker uses `aria-step` to indicate which is the current active step.
+### Best practices
+
+Stepped trackers use `aria-step` to indicate the current active step.
```

Comment:
This doesn't seem like a best practice; it's actually describing the implementation.

The sad part is that the original content is wrong. It actually uses `aria-current="step"`